### DISCUSSION TITLE
How can we customized the "add" button when having a list?

### DISCUSSION THREAD

**Discussion by fablexis** [vueform/vueform#161] [Questions]
Checking the docs I saw that the `list` type has a `controls` key to enable the "add" and "remove" features for the list elements, even though, when having `controls: {add: true}`, a default button is displayed to provide a way to add more elements to the list... at this point, I wonder how I can customize that button... let's say I don't want a button but only a simple text with a hyperlink style, how can we customize or specify that in our `schema` object? do we have a way to specify/modify that default behavior? currently what I saw was only an option to change that button text, but I did not find any reference to modify the styles of that button

![image](https://github.com/vueform/vueform/assets/7627658/b1c3883b-e377-4189-9ff6-4f1d099df0b7)

**Comment by adamberecz** [comment]
See my answer here: https://github.com/vueform/vueform/discussions/160#discussioncomment-8633716

**Reply by fablexis** [reply]
thanks for the reference!

I wonder how we can define the template in an object notation like the structure of the `schema`, most of the examples show the HTML reference instead of the alternative using an object for the schema
